go/internal/feast/registry.go

```diff
@@ -28,9 +30,10 @@ type Registry struct {
 	cachedFeatureViews         map[string]map[string]*core.FeatureView
 	cachedOnDemandFeatureViews map[string]map[string]*core.OnDemandFeatureView
 	cachedRequestFeatureViews  map[string]map[string]*core.RequestFeatureView
-
+	cachedRegistry             *core.Registry
 	cachedRegistryProtoCreated time.Time
```

Comment:
Maybe this should be `cachedRegistryProtoUpdated`? Since it's when the cached registry was last updated.

Reply:
Done.